src/c++/perf_analyzer/genai-perf/README.md

```diff
@@ -345,14 +345,16 @@ Show the help message and exit.
 ##### `-m <list>`
 ##### `--model <list>`
 
-The name of the model to benchmark. (default: `None`)
+The names of the models to benchmark.
```

Comment:
Hey! Thanks for catching this. 
I should  have updated this with the previous PR.
🦸🏼